src/main/webapp/login.html

```diff
@@ -1,22 +1,34 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>altruism-ator</title>
+    <title>Login</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-      <header>
-        <div class="container">
-            <nav>
-                <ul>
-                    <li class="nav-item"><a href="index.html">Home</a></li>
-                    <li class="nav-item"><a href="profile.html">Profile</a></li>
-                    <li class="nav-item"><a href="createYourOwn.html">Create your own</a></li>
-                    <li class="nav-item"><a href="chat.html">Chat</a></li>
-                </ul>
-            </nav>
-        </div>
-      </header>
+    <header>
+      <div class="container">
+        <nav>
+          <ul>
+            <li class="nav-item"><a href="index.html">Home</a></li>
+            <li class="nav-item"><a href="profile.html">Profile</a></li>
+            <li class="nav-item"><a href="createYourOwn.html">Create your own</a></li>
+            <li class="nav-item"><a href="chat.html">Chat</a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="loginForm">
+        <form action="/login" method="POST">
+          <label for="username">Username:</label>
+          <input type="text" name="username" required><br><br>
+          <label for="password">Password:</label>
+          <input type="password" name="password" required><br><br>
+          <input type="submit">
+        </form>
+        <a href="signUp.html">
+          <button>Sign Up</button>
+        </a>
+      </div>
+    </header>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Add a trailing line.

Reply:
Interesting, I see a trailing line when I open that file.